apps/content-publishing-api/src/controllers/v1/development.controller.v1.ts

```diff
@@ -64,20 +62,6 @@ export class DevelopmentControllerV1 {
     return job;
   }
 
-  @Get('/asset/:assetId')
```

Comment:
Any particular reason we don't want to keep this endpoint for dev/debug?

Reply:
So this was the only reference to IPFSService used in the content-publishing-api and by removing it we don't need have a dependency on IPFS module. Also this is not an important dev endpoint since we can also get the same info from ipfs dashboard